src/main/flight/mixer.c

```diff
@@ -607,9 +621,25 @@ FAST_CODE_NOINLINE void mixTable(timeUs_t currentTimeUs)
     // Find min and max throttle based on conditions. Throttle has to be known before mixing
     calculateThrottleAndCurrentMotorEndpoints(currentTimeUs);
 
+    static bool turtleModeStarted = false;
     if (isFlipOverAfterCrashActive()) {
-        applyFlipOverAfterCrashModeToMotors();
+        static float tiltAngleAtStart = 1.0f;
+        float flipAngleModifier = 1.0f;
+#ifdef USE_ACC
```

Comment:
There is no check for acc availability here, yet. `#ifdef USE_ACC` is just a macro. It cannot detect acc availability. Wrap everything in an if-statement like this:
```suggestion
#ifdef USE_ACC
        if (sensors(SENSOR_ACC) && accIsCalibrationComplete()) {
            ...
        }
#endif // USE_ACC
```

Reply:
Hmm.  I can do that, though I need to add sensors/accelerometer.h.
However, the code handles having Acc disabled without any apparent bad stuff happening - it won't turn the motors off, obviously, but it will still rotate at the semi-controlled turn rate.
I see that `getCosTiltAngle` returns `rMat.[2][2]`.  Could it be that when Acc is disabled, the `rMat` function returns the same value every time it is called, leading to the same `getCosTiltAngle` value every time?  If that is the case, our function just will always think the quad never changed attitude, and we won't need an additional check.
What do you think?  I can definitely add the `if (sensors(SENSOR_ACC)) {` check if you think it is necessary.

Reply:
So inverted yawing still works? Then I made a mistake by thinking about the angle attenuation the wrong way (as an absolute value instead of a difference)

Reply:
`#ifdef USE_ACC` doesn't check for an acc. Macros are not part of the runtime code. Without the if-statement this code can run even if there is no acc on the FC.

Reply:
OK, that's strange... now it does work.  Confusing!  Seems OK now - thanks again.